src/models/payments.js

```diff
@@ -25,6 +25,10 @@ function * settle (sourceTransfer, destinationTransfer, config, ledgers) {
           code: 'S00',
           name: 'Bad Request',
           message: 'destination transfer failed: ' + err.message
+        } : (err.name === 'NotAcceptedError') ? {
```

Comment:
That is true. Plugins don't have an explicit "Insufficient Source Amount" error right now (it falls under `NotAcceptedError`) -- maybe they should. See: https://github.com/interledger/rfcs/pull/281